src/components/page-layout/components/layouts/example/content.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ExamplesPage from '../../../../examples-page/examples-page';
+import CardContainer from '../../../../card-container/card-container';
+import Card from '../../../../card/card';
+
+export default class LayoutExamples extends React.PureComponent {
+  renderThumbnail = (thumbnail, AppropriateImage) => {
+    // if thumbnail has an image extenstion, handle the image
+    if (/\.png|jpeg|jpg|gif$/.exec(thumbnail)) {
+      return (
+        <div
+          className="absolute top bottom w-full h-full round"
+          style={{
+            background: `url(${thumbnail})`,
+            backgroundSize: '100%'
+          }}
+        />
+      );
+    } else if (AppropriateImage && thumbnail) {
+      // if thumbnail has AppropriateImage function and thumbnail exists, handle with AppropriateImage
+      return (
+        <AppropriateImage
+          style={{ borderRadius: '4px' }}
+          imageId={thumbnail}
+          background={true}
+        />
+      );
+    } else {
+      // else return nothing
+      return undefined;
```

Comment:
If it's an invalid imageId, then the content area will break:

![image](https://user-images.githubusercontent.com/2180540/90524572-3dd6dc00-e13c-11ea-9d14-78d3da8386e0.png)

If it's an invalid image file, then it will appear as a broken image.